testdata/project-v2/main.go

```diff
@@ -37,8 +37,8 @@ var (
 
 func init() {
 
-	crewv1.AddToScheme(scheme)
-	corev1.AddToScheme(scheme)
+	_ = crewv1.AddToScheme(scheme)
```

Comment:
Shouldn't these errors be checked? I've seen `Must(crewv1.AddToScheme(scheme))` in k8s codebases

where 
```go
func Must(err error) {
  if err != nil {
    panic(err)
  }
}
```

Reply:
Ideally yes and that is a correct resolution for the error reported by `AddToScheme`. Implementing  suggested fix will require a bit of work/refactoring on how we generate the code fragments for `AddToScheme` calls and don't want to hold this PR for that to happen. We can do that in a separate PR (may be with a PR that runs `golangci-lint run` instead of `go vet` for generated project as suggested above).

Reply:
created an issue https://github.com/kubernetes-sigs/kubebuilder/issues/791 to track it.